beacon-chain/rpc/eth/light-client/helpers_test.go

```diff
@@ -26,6 +26,11 @@ func createNonEmptyFinalityBranch() [][]byte {
 }
 
 func TestIsBetterUpdate(t *testing.T) {
+
+	params.SetupTestConfigCleanup(t)
+	config := params.MainnetConfig()
+	params.OverrideBeaconConfig(config)
```

Comment:
This was a bit strange. I had to manually set the `params` to `MainnetConfig` otherwise `bazel test` would fail. It turns out that when I run the `handlers-test.go` immediately before `helpers-test.go` this test fails. and bazel runs them alphabetically. I could not figure out what the problem in `handlers-test` was, so for now I just manually set the params here.

Reply:
You have to add `params.SetupTestConfigCleanup(t)` at the beginning of every test, otherwise the config change will be visible between tests